sign.js

```diff
@@ -219,6 +219,9 @@ var signAsync = module.exports.signAsync = function (opts) {
       var promise
       if (opts.identity) {
         debuglog('`identity` passed in arguments.')
+        if (opts.skipIdentityValidation) {
```

Comment:
I prefer camel-case, but you are right — I will rename to `identity-validation` for consistency.